### PR TITLE
Chore/merge main@2fe90d8

### PR DESCRIPTION
**Pull request by julien-deramond**

### CODE REVIEW

File: js/tests/browsers.js [hunk 1]
```diff
@@ -51,13 +51,6 @@ const browsers = {
     browser: 'Firefox',
     browser_version: 'latest'
   },
-  EsrWin10: {
```
Comment (julien-deramond) [inline js/tests/browsers.js:54]: This one was only in Boosted. I chose to remove it to have the same file than Bootstrap.

Comment (Lausselloic) [reply js/tests/browsers.js:54]: But ESR is the one used inside Orange 🤔

**Review by louismaximepiton — APPROVED** [review]
Many little corrections to add such as `js/tests/tab` line 117 -> `</button>` tag instead of `</a>`, I'll provide patches to Bootstrap directly.

File: js/tests/browsers.js (hunk 1 above)
Comment (julien-deramond) [reply js/tests/browsers.js:54]: OK: verified in the history of both Boosted and Bootstrap. Thanks for the info, I've put it back in this PR surrounded by a `Boosted mod` comment to highlight it via 45207e3c0b0bae8c76689c911efdff836e5eb8dc

**Pull request merged**